### ISSUE TITLE
iMacPro sync fans

### ISSUE THREAD

**Issue by kleuter** [crystalidea/macs-fan-control#31]
>  could you ad a function linking control of both iMac Pro fans together? I just tested the OSX version. It doesn't make much sense having different individual speeds, and you can hear the fans are thrown off by having different speeds.
@Carl37

**Comment by kleuter** [comment]
I don't have this machine, are you sure that mac always spins both fans identically ?

**Comment by Carl37** [comment]
Yes, both fans should always be in sync. Just take a look at a tear down of the iMac Pro. The fans are situated next to each other, and are part of the same system/air flow. Ramping them up and down out of sync (ie. one at a time) clearly affects them negatively and they rotate unevenly. Having the fans run in sync is definitely preferable.

**Comment by kleuter** [comment]
Thanks, tomorrow I'll have this machine to test.

**Comment by kleuter** [comment]
True. The mac always spins both fans (either because of CPU or GPU activity), checked with couple of benchmarks. Thanks you.

**Comment by kleuter** [comment]
<img width="1023" alt="screen shot 2018-02-22 at 23 32 18" src="https://user-images.githubusercontent.com/2600624/36568005-c60d5dcc-1828-11e8-8160-58522e5c2291.png">

**Comment by Carl37** [comment]
Nice! I missed that one. Is this beta version with synced fans available for download? I'm using your program a lot these days. 
Thanks!

**Comment by kleuter** [comment]
Please re-download the [beta](https://www.crystalidea.com/downloads/macsfancontrol_beta.zip) to test

**Comment by Carl37** [comment]
Seems to be working great, thanks! I see you added more temperature sensors also. How does the min-max temperature range work? So if the machine's temperature exceeds the range set in your program, the fan control will revert back to the machine's own SMC? Or is it max fan speed when the max temp value is reached? Just curious... :) 

Thanks again, greatly appreciated!

**Comment by kleuter** [comment]
min/max values are read from the SMC

**Comment by Carl37** [comment]
Sorry if I'm being obtuse, but does that mean if I set min/max values in your program, they substitute the SMC values (min. value is turning up the fan speed, max. value is fans at max speed)? I'm a understanding this correctly?
Thanks.

**Comment by Carl37** [comment]
So max temperature value in Macs Fan Control is fans at max speed? Am I understanding this correctly? If this function works properly, it's a huge deal. Having to turn on a high static fan speed in OSX, and then booting into Bootcamp isn't exactly optimal. Thanks!

**Comment by Carl37** [comment]
From testing it I see max temperature value in Macs Fan Control turns the fans up to max speed. Thanks.

PS: The min/max function works in OSX, but if I boot into Bootcamp/Windows the fans will be stuck at whatever RPM they're at when I rebooted. The min/max values doesn't work when booted into Windows. Don't know if that's useful or not, but thought I'd mention it anyway.
